Java/libraries/recognizers-text-number-with-unit/src/main/java/com/microsoft/recognizers/text/numberwithunit/extractors/BaseMergedUnitExtractor.java

```diff
@@ -30,19 +30,22 @@ public BaseMergedUnitExtractor(INumberWithUnitExtractorConfiguration config) {
         }
     }
 
+    @SuppressWarnings("unchecked")
     private List<ExtractResult> mergeCompoundUnits(String source) {
         List<ExtractResult> ers = new NumberWithUnitExtractor(config).extract(source);
         mergePureNumber(source, ers);
 
-        if(ers.size() == 0) return ers;
+        if (ers.size() == 0) {
+            return ers;
+        }            
 
         List<ExtractResult> result = new ArrayList<>();
         int[] groups = new int[ers.size()];
         groups[0] = 0;
         for (int idx = 0; idx < ers.size() - 1; idx++) {
-            if (!ers.get(idx).type.equals(ers.get(idx + 1).type) &&
-                    !ers.get(idx).type.equals(Constants.SYS_NUM) &&
-                    !ers.get(idx + 1).type.equals(Constants.SYS_NUM)) {
+            if (!ers.get(idx).type.equals(ers.get(idx + 1).type)
+                &&  !ers.get(idx).type.equals(Constants.SYS_NUM)
```

Comment:
Nit: we use the convention of keeping operators at the end of the previous line, not the start of the next. Could you change the linting rules for it?
BTW, is it possible to set it to not change spacing/indentation of making long lines multi-lines? For example, in this case, the original indentation was fine.